typed-racket-lib/typed-racket/rep/type-rep.rkt

```diff
@@ -124,7 +140,8 @@
                 [args exact-nonnegative-integer?]
                 [struct? boolean?])
   #:base
-  [#:custom-constructor
+  [#:custom-constructor/contract
```

Comment:
Right, but in almost all cases the "obvious" contract is the right one, and we're just duplicating.